.github/workflows/pr.yml

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+
+name: CI
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
So this is running in addition to the `main.yml`?

Reply:
No it shouldn't be...! I updated.